scripts/rosa/delete-rosa-clusters.sh

```diff
@@ -69,27 +69,43 @@ delete_load_balancers() {
         tags=$(aws --region "$AWS_DEFAULT_REGION" elbv2 describe-tags --resource-arns "$lb_arn" --query "TagDescriptions[*].Tags[?Key=='$LB_TAG_KEY'&&Value=='$lb_tag_value'].Value" --output text)
         
         if [[ "$tags" == "$lb_tag_value" ]]; then
-
             echo "[INFO] Deleting ELBv2 Load Balancer with ARN: $lb_arn"
             aws --region "$AWS_DEFAULT_REGION" elbv2 delete-load-balancer --load-balancer-arn "$lb_arn"
-
-            echo "[INFO] Deleting Target Groups associated with Load Balancer ARN: $lb_arn"
-            delete_target_groups_for_lb "$lb_arn"
         fi
     done
 }
 
-delete_target_groups_for_lb() {
-    local lb_arn=$1
-    local tg_arns
+delete_target_groups() {
+    TODAY=$(date -u -d '24 hours ago' +"%Y-%m-%dT%H:%M:%SZ")
+
+    ALL_TARGET_GROUPS=$(aws elbv2 describe-target-groups --region "$REGION" --query 'TargetGroups[*].TargetGroupArn' --output text)
```

Comment:
```suggestion
    ALL_TARGET_GROUPS=$(aws elbv2 describe-target-groups --region "$AWS_DEFAULT_REGION" --query 'TargetGroups[*].TargetGroupArn' --output text)
```
and similarly in other occurrences of this env var in this function.